cfme/infrastructure/provider/rhevm.py

```diff
@@ -18,6 +18,12 @@ def view_value_mapping(self):
                 'verify_tls': getattr(self, 'verify_tls', None),
                 'ca_certs': getattr(self, 'ca_certs', None)}
 
+    def delete_tls_info(self):
```

Comment:
First impression its a little odd looking, but I much prefer this to having provider-specific modifications in the `BaseProvider.create()` method. This means a provider instance will have a default endpoint with key/value pairs relevant for the appliance version, and view_value_mapping includes them when they're there.